### ISSUE TITLE
Query string weirdness

### ISSUE THREAD

**Issue by building39** [gar1t/modlib#1]
When I run the sample code with a query string like "?parm1:1;parm2:2;parm3", I get this kind of response:

[{"parm1:1", []}, "parm2:2", []}, {"parm3", []}]

rather than the expected:

[{"parm1", [1]}, "parm2", [2]}, {"parm3", []}]

**Comment by gar1t** [comment]
This code is lifted from Mochiweb and it does appear to split in ';' which
is surprising.

But your example uses ':' as the separator of the field name and value. It
should be '='. The separator between the fields should be '&'.

Use this instead:

?parm1=1&parm2=2&parm3

**Comment by building39** [comment]
Yes, of course you're right. The ":" is the separator for riak search predicates. I really should go to bed earlier...
